lib/internal/Magento/Framework/Test/Unit/Data/CollectionTest.php

```diff
@@ -57,6 +57,25 @@ public function testWalk()
         );
     }
 
+    /**
+     * Ensure that getSize works correctly with clear
+     *
+     * @throws \Exception
```

Comment:
A don't think we need such notation in tests.

Reply:
@orlangur thanks, fixed